examples/nD_pyramid.py

```diff
@@ -2,26 +2,20 @@
 Displays an image pyramid
 """
 
-from skimage import data
-from skimage.util import img_as_ubyte
-from skimage.color import rgb2gray
 from skimage.transform import pyramid_gaussian
 import napari
 import numpy as np
 
 
-# create pyramid from astronaut image
-astronaut = data.astronaut()
-base = np.tile(astronaut, (3, 3, 1)).astype('float')
-base = np.round(np.array([base * (16 - i) / 16 for i in range(16)])).astype(
-    np.uint8
-)
+# create pyramid from random data
+base = np.random.random((1536, 1536))
+base = np.array([base * (8 - i) / 8 for i in range(8)])
```

Comment:
what's going on here???

Reply:
I wanted to introduce some variation into the random image, which I think is better than the tiled astronaut. People found it confusing what that represented. We can still improve this further, but I think it is fine for now

Reply:
What I don’t understand is why this improves on just the random image in any way. Why go through the trouble of multiplying a random image by random numbers?

Reply:
Oh, wait, — does this add a third dimension? And if so, why not just add a third dimension to the random call?

Reply:
(I agree that the tiling was confusing, btw)